website/docs/r/kmip_secret_backend.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "vault"
+page_title: "Vault: vault_kmip_secret_backend resource"
+sidebar_current: "docs-vault-resource-kmip-secret-backend"
+description: |-
+  Provision KMIP Secret backends in Vault.
+---
+
+# vault\_kmip\_secret\_backend
+
+Manages KMIP Secret backends in a Vault server. See the [Vault
```

Comment:
Might want to mention that this feature requires Enterprise.

Reply:
Added in 3686e05